Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Node.h

```diff
@@ -1098,7 +1098,7 @@ namespace ScriptCanvas
                 slotConfiguration.SetType(Data::FromAZType<AZStd::decay_t<ResultType>>());
                 slotConfiguration.SetConnectionType(ConnectionType::Output);
 
-                node.AddSlot(slotConfiguration);
+                AZ_VerifyError("ScriptCanvas", node.AddSlot(slotConfiguration).IsValid(), "Node failed to add a required Data Out slot");
```

Comment:
Is there anything to add to the error message for how someone might go about debugging or fixing the error?